acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/steps/ContentSteps.java

```diff
@@ -288,4 +286,42 @@ public void iAddAnEmptyRelatedLinkField() throws Throwable {
         contentPage.getRelatedLinksSection().addRelatedLinkField();
     }
 
+
+    @Given("^I have a released publication flagged as upcoming$")
+    public void iHaveAReleasedPublicationFlaggedAsUpcoming() throws Throwable {
+        final Publication publication = TestDataLoader.loadReleasedUpcomingPublication().build();
+        testDataRepo.setCurrentPublication(publication);
+    }
+
+    @When("^I view the publication$")
+    public void iViewThePublication() throws Throwable {
+        consumablePublicationPage.open(testDataRepo.getCurrentPublication());
+    }
+
+    @Then("^Title is visible$")
+    public void titleIsVisible() throws Throwable {
+        assertThat("Title is visible.",
```

Comment:
This asserts more than visibility; should it?

Reply:
Agreed, I'll change it to 'shown'.

Reply:
Done.